ITISFoundation/osparc-ops-environments
# BUGFIX: Dalco Staging pgsql-query-exporter cannot connect to postgres

## What do these changes do
These changes fixes a wrongly configured password for `pgsql-query-exporter` on `dalco staging`.  Previously, we couldn't configure multiple postgres password for `pgsql-query-exporter` (sometimes, we want it to reach to different postgres instances).

These changes introduce opportunity to set different passwords for different stacks monitored by `pgsql-query-exporter`. The config files are updated accordignly in `osparc-infra`.

## Related Issues
https://github.com/ITISFoundation/osparc-ops-environments/issues/262

## How to test
Deploy dalco prod and check that `pgsql-query-exporter` can connect to staging postgres
